helm/ffc-ahwr-frontend/values.yaml

```diff
@@ -25,8 +25,6 @@ containerSecret:
 container:
   applicationRequestQueueAddress: ffc-ahwr-application-request
   applicationResponseQueueAddress: ffc-ahwr-application-response
-  fetchApplicationRequestQueueAddress: ffc-ahwr-fetch-application-request
```

Comment:
No such queues.

Reply:
Yes I think I added it and forgot to remove it 🤦‍♂️